### PR TITLE
Fix issues with note

### PR DESCRIPTION
**Pull request by AprupKale**
Fix all issues found with note inlcuding:
- [x] Note not being updated in the model
- [x] Note command without note parameter error

### CODE REVIEW

**Comment by AprupKale** [comment]
@tim-pipi thanks for catching the bugs. I have made the fixes!

File: src/main/java/seedu/letsgethired/logic/commands/NoteCommand.java
Comment (tim-pipi) [inline src/main/java/seedu/letsgethired/logic/commands/NoteCommand.java:1]:
```
private String generateSuccessMessage(InternApplication personToEdit) {
        String message = !note.value.isEmpty() ? MESSAGE_ADD_NOTE_SUCCESS : MESSAGE_DELETE_NOTE_SUCCESS;
        return String.format(message, Messages.format(personToEdit));
    }
```
Perhaps this is better to follow the convention of success message of other commands as well!

**Review by tim-pipi — CHANGES_REQUESTED** [review]
LGTM overall! Thanks for the changes, could you add the test cases as well to pass the codecov? Thank you for the hard work Aprup!

**Review by tim-pipi — APPROVED** [review]

**Comment by tim-pipi** [comment]
LGTM! Thanks for the changes! Will merge this PR! 👍

**Pull request merged**